### ISSUE TITLE
Excel Test Report

### ISSUE THREAD

**Issue by ty-desj** [StatCan/tass-ssat#163]

**Comment by ty-desj** [comment]
@DominicParent @Pswenne I have included a few of the data views/charts we discussed the other day. Any other suggestions for what to include in the default excel report?

**Comment by DominicParent** [comment]
Can't think of anything else to include by default. Chris might have some ideas.

**Comment by ty-desj** [comment]
After reviewing some of the openpyxl features and our own JSON result document. I think it might be beneficial to refactor the json output to better organize/display the results in a way that is more easily readable by both people and the machine. 

I've created a new issue to track this change separately from the Excel report function. It should be completed before a full Excel reporting implementation to minimize risk of breaking changes to the Excel reporting. #165

**Comment by DominicParent** [comment]
Further discussion points to creating a small Excel report with filtered columns.

Should also update the JSON report file to have some sort of default status for steps that haven't been run yet.
